functionalTest/test/pageobjects/dashboardPage.js

```diff
@@ -40,6 +31,327 @@ class DashboardPage extends Page {
   get percentageCases() {
     return $("#percentageCases");
   }
+  // Heatmap selector
+  // Health Boards
+  get healthBoard() {
+    return $("#healthBoards");
+  }
+  get healthBoardTitleText() {
+    return $("//div[text()='HEALTH BOARDS']");
+  }
+  get healthBoardSubheadingCount() {
+    return $("//div[@class='subheading'][text()='14 Boards']");
+  }
+  get healthBoardTotalCasesText() {
+    return $("//div[text()='TOTAL CASES']");
+  }
+  get healthBoardTotalCasesCount() {
+    return $("//div[@class='subheading'][text()='34760']");
+  }
+  get healthBoardTotalDeathsText() {
+    return $("//div[text()='TOTAL DEATHS']");
+  }
+  get healthBoardTotalDeathsCount() {
+    return $("//div[@class='subheading'][text()='2533']");
+  }
+  get healthBoard01Name() {
+    return $("//td[text()='Ayrshire & Arran']");
+  }
+  get healthBoard02Name() {
+    return $("//td[text()='Borders']");
+  }
+  get healthBoard03Name() {
+    return $("//td[text()='Dumfries & Galloway']");
+  }
+  get healthBoard04Name() {
+    return $("//td[text()='Fife']");
+  }
+  get healthBoard05Name() {
+    return $("//td[text()='Forth Valley']");
+  }
+  get healthBoard06Name() {
+    return $("//td[text()='Grampian']");
+  }
+  get healthBoard07Name() {
+    return $("//td[text()='Greater Glasgow & Clyde']");
+  }
+  get healthBoard08Name() {
+    return $("//td[text()='Highland']");
+  }
+  get healthBoard09Name() {
+    return $("//td[text()='Lanarkshire']");
+  }
+  get healthBoard10Name() {
+    return $("//td[text()='Lothian']");
+  }
+  get healthBoard11Name() {
+    return $("//td[text()='Orkney']");
+  }
+  get healthBoard12Name() {
+    return $("//td[text()='Shetland']");
+  }
+  get healthBoard13Name() {
+    return $("//td[text()='Tayside']");
+  }
+  get healthBoard14Name() {
+    return $("//td[text()='Western Isles']");
+  }
+  get healthBoard01Count() {
+    return $("//tbody/tr[2]/td[2]");
+  }
+  get healthBoard02Count() {
+    return $("//tbody/tr[3]/td[2]");
+  }
+  get healthBoard03Count() {
+    return $("//tbody/tr[4]/td[2]");
+  }
+  get healthBoard04Count() {
+    return $("//tbody/tr[5]/td[2]");
+  }
+  get healthBoard05Count() {
+    return $("//tbody/tr[6]/td[2]");
+  }
+  get healthBoard06Count() {
+    return $("//tbody/tr[7]/td[2]");
+  }
+  get healthBoard07Count() {
+    return $("//tbody/tr[8]/td[2]");
+  }
+  get healthBoard08Count() {
+    return $("//tbody/tr[9]/td[2]");
+  }
+  get healthBoard09Count() {
```

Comment:
There's a lot of unnecessary replication here - we can trim this down a lot

Reply:
Noted :)